code/modules/mob/living/carbon/human/species.dm

```diff
@@ -845,30 +857,33 @@
 	warcries = list(MALE = "male_warcry", FEMALE = "female_warcry")
 	special_death_message = "You have been shut down.<br><small>But it is not the end of you yet... if you still have your body, wait until somebody can resurrect you...</small>"
 
-
-/datum/species/synthetic/handle_post_spawn(mob/living/carbon/human/H)
+/datum/species/synthetic/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
 	. = ..()
 	var/datum/atom_hud/AH = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED_SYNTH]
 	AH.add_hud_to(H)
-
-/mob/living/carbon/human/species/synthetic/binarycheck(mob/H)
-	return TRUE
+	H.health_threshold_crit = -100
```

Comment:
No synthethic has been powercreped enough

Reply:
Fair enough, can skeletons keep it though please? I'd thank you a skele-TON if you let me.

Reply:
CL and synthetics fixed